app/assets/javascripts/analytics/error-tracking.js

```diff
@@ -0,0 +1,24 @@
+// Extension to track errors using google analytics as a data store.
+(function() {
+
+    "use strict";
+    var trackJavaScriptError = function (e) {
+      var errorSource = e.filename + ': ' + e.lineno;
+      _gaq.push([
+        '_trackEvent',
+        'JavaScript Error',
+        e.message,
+        errorSource,
+        true //nonInteractive so bounce rate isn't affected
```

Comment:
You're missing the 4th event parameter, which is an integer "value" (https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide#Values).  The nonInteractive flag comes after that.

Elsewhere we've just set this to 1: eg https://github.com/alphagov/static/blob/master/app/assets/javascripts/browser-check.js#L23